infra/libfuzzer-pipeline.groovy

```diff
@@ -22,12 +22,11 @@ def call(body) {
     body()
 
     // Mandatory configuration
-    def dockerfile = config["dockerfile"]
-    assert dockerfile : "dockerfile should be specified"
     def gitUrl = config["git"]
     assert gitUrl : "git should be specified"
 
     // Optional configuration
+    def dockerfile = config["dockerfile"]
```

Comment:
let's move it one line down and compute the default right here.

"oss-fuzz/$projectName/Dockerfile".

No need to have absolute file name as well.